Sprint-1/implement/max.js

```diff
@@ -1,4 +1,18 @@
+//I will do for loop to iterate every element
+//I have to use that filters number only the array to number
+//check every index is whether greater than the next index
 function findMax(elements) {
+    let smallestNumber = -Infinity;
+    for(let index=0; index < elements.length; index++){
+        const currentNumber = elements[index];
+        if (!isNaN(currentNumber) && currentNumber > smallestNumber){
```

Comment:
Because of the way JS compares values of different types, some non-number-type values can pass this test.
Here are some examples,

```
findMax(["300", 2])
findMax(["300", "4"])
findMax([[100], 4, [200]])
```